.github/workflows/terraform-plan-apply.yml

```diff
@@ -302,6 +307,37 @@ jobs:
           az storage blob delete-batch --connection-string ${ui_storage_conn_string} --source "\$web" --pattern "*"
           az storage blob upload-batch --connection-string ${ui_storage_conn_string} --source mts-trial-ui --destination '$web'
 
+      # Download init service progress log file from azure file share and validate success
+      - name: Validate Init service progress
+        # if: github.ref == 'refs/heads/main'
+        run: |
+          #can't use errexit and pipefail flags here becasue tail will fail if the file doesn't exists
+          set -o nounset
+          #set -o xtrace
```

Comment:
delete line (if should be commented)

Reply:
I left this remark on purpose, so others won't add by mistake errexit or pipefail flags cause I don't want the code to exit on error.
do u think I should remove it anyway?